s3/libirods_s3.cpp

```diff
@@ -1491,13 +1530,19 @@ static void mpuWorkerThread (
             }
         } while ((partData.status != S3StatusOK) && S3_status_is_retryable(partData.status) && (++retry_cnt <= retry_count_limit));
         if (partData.status != S3StatusOK) {
-            msg.str( std::string() ); // Clear
-            msg << "[resource_name=" << resource_name << "] " << __FUNCTION__ << " - Error putting the S3 object: \"" << g_mpuKey << "\"" << " part " << seq;
+
+            auto msg = fmt::format("[resource_name={}] {} - Error putting the S3 object: \"{}\" part {}",
+                    resource_name,
+                    __FUNCTION__,
+                    g_mpuKey,
+                    seq);
+
             if(partData.status >= 0) {
-                msg << " - \"" << S3_get_status_name(partData.status) << "\"";
+                msg += fmt::format(" - \"{}\"", S3_get_status_name((S3Status)partData.status));
             }
-            result = ERROR( S3_PUT_ERROR, msg.str() );
-            rodsLog( LOG_ERROR, msg.str().c_str() );
+
+            result = ERROR( S3_PUT_ERROR, msg );
```

Comment:
The value of `result` can be assigned to `g_mpuResult` directly.

Not required by any means.

Reply:
done